src/styles/style.css

```diff
@@ -0,0 +1,7 @@
+/* @import url(blocks/body.css);
+@import url(blocks/stars.css);
+@import url(blocks/stars__star.css); */
```

Comment:
The commented-out import statements are unnecessary and can be removed to clean up the code.